features/step_definitions/tag_steps.rb

```diff
@@ -153,8 +153,37 @@
   end
 end
 
+Given(/^the following unsorted tag exists$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:unsorted_tag, hash)
+  end
+end
+
+Given(/^the following typed tags exists$/) do |table|
+  table.hashes.each do |hash|
+    type = hash["type"].classify.constantize
+    hash.delete("type")
+    FactoryGirl.create(type, hash)
+  end
+end
+
 ### WHEN
 
+When /^I check the canonical wrangling tag "([^"]*)"$/ do |tagname|
+  tag = Tag.find_by_name(tagname)
+  check("canonicals_#{tag.id}")
+end
+
+When /^I select "([^"]*)" for "([^"]*)" tag$/ do |type, tagname|
+  tag = Tag.find_by_name(tagname)
+  select(type, :from => "tags[#{tag.id}]")
+end
+
+When /^I check the wrangling tag "([^"]*)"$/ do |tagname|
```

Comment:
And is this the checkbox in front of the tag on pages like http://test.archiveofourown.org/tag_wranglings?show=fandoms and http://test.archiveofourown.org/tags/Dallas%20(2012)/wrangle?show=characters&status=unwrangled?

If so: `I check the tag "([^"]*)" for mass wrangling` or `I check the mass wrangling option for "([^"]*)"` or `I select the tag "([^"]*)" for mass wrangling`